homeassistant/components/shelly/strings.json

```diff
@@ -138,21 +124,27 @@
       }
     },
     "sensor": {
+      "gas_detected": {
+        "state": {
+          "none": "None",
+          "mild": "Mild",
+          "heavy": "Heavy",
+          "test": "Test"
```

Comment:
We should add translations for `options` attribute like here https://github.com/home-assistant/core/pull/140662